thrift.js

```diff
@@ -254,8 +254,7 @@ Thrift.prototype._compile = function _compile(defs) {
 };
 
 Thrift.prototype.compileInclude = function compileInclude(def) {
-    if (def.id.lastIndexOf('./', 0) === 0 ||
-        def.id.lastIndexOf('../', 0) === 0) {
+    if (def.id.indexOf('/') !== 0) {
```

Comment:
Can be faster by saying search from index 0 back toward 0: `def.id.lastIndexOf('/', 0) !== 0`